roles/openshift_aws/tasks/provision_masters.yml

```diff
@@ -0,0 +1,34 @@
+---
+- import_tasks: vpc_and_subnet_id.yml
+
+- import_tasks: provision_ec2_facts.yml
+
+- name: include master instance creation
+  import_tasks: provision_ec2.yml
+
+- name: fetch newly created instances
+  ec2_instance_facts:
+    region: "{{ openshift_aws_region }}"
+    filters:
+      "tag:clusterid": "{{ openshift_aws_clusterid }}"
+      "tag:host-type": "master"
+      instance-state-name: running
+  register: instancesout
+  retries: 20
+  delay: 3
+  until: instancesout.instances|length > 0
```

Comment:
I'm not sure how likely it is, but it seems possible one could go 'running' before the others and we don't have the proper amount here.  Need to check for specific length of results.